### PR TITLE
EXPERIMENTAL - OOC Cross Communications System

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

OOC messages are sent across servers using send2otherserver, DNM.

In preparation for something.

## Changelog
:cl:
server: This codebase now supports sending cross communications OOC messages to other servers.
/:cl:

### CODE REVIEW

File: config/comms.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Communication key for receiving data through world/Topic(), you don't want to give this out
-#COMMS_KEY default_pwd
+COMMS_KEY skyratserver
```
Comment (unit0016) [inline config/comms.txt]: Probably best to leave this at default for the repo.

**Comment by NullDagaf** [comment]
I'm.. not sure why you'd need cross-server OOC. If you want to announce something, just write it in the discord announcements, yeah?

**Comment by Gandalf2k15** [comment]
Dont care anymore

**Comment by NullDagaf** [comment]
> Dont care anymore

Based

**Pull request closed without merging**